src/controllers/hotkeys/HotkeyController.cpp

```diff
@@ -516,7 +516,7 @@ void HotkeyController::addDefaults(std::set<QString> &addedHotkeys)
                             std::vector<QString>(), "reopen split");
 
         this->tryAddDefault(addedHotkeys, HotkeyCategory::Window,
-                            QKeySequence("Ctrl+H"), "toggleLocalR9K",
+                            QKeySequence("Ctrl+."), "toggleLocalR9K",
```

Comment:
This should be [`#ifdef Q_OS_MACOS`](https://doc.qt.io/qt-6/qtsystemdetection.html#Q_OS_MACOS)d.